packages/dialtone-css/package.json

```
@@ -51,8 +51,7 @@
   "scripts": {
     "lint": "run-s lint:code lint:library",
     "lint:code": "eslint '**/*.{js,cjs,vue}'",
-    "lint:library": "run-s lesshint stylelint",
-    "lesshint": "pnpm exec lesshint 'lib/build/less/utilities'",
+    "lint:library": "run-s stylelint",
```

Not necessary to keep `run-s` as there's only one command left to run.

Apply the suggestion and remove `stylelint` command on line 55.

```suggestion
    "lint:library": "pnpm exec stylelint 'lib/build/less/**/*.less'",
```